react/table/src/components/ConferenceTableContainer.js

```diff
@@ -139,9 +144,56 @@ class ConferenceTableContainer extends Component {
     });
   }
 
+  async handleDelete(item) {
+    const { onDelete, keycloak } = this.props;
+    const authenticated = keycloak.initialized && keycloak.authenticated;
+
+    if (authenticated) {
+      try {
+        await apiConferenceDelete(item.id);
+        onDelete(item);
+        this.dispatch({ type: DELETE, payload: { id: item.id } });
+      } catch (err) {
+        this.handleError(err);
+      }
+    }
+  }
+
+  handleConfirmationDialogAction(action, item) {
+    switch (action) {
+      case ConfirmationDialogTrigger.CONFIRM: {
+        this.handleDelete(item);
+        break;
+      }
+      default:
+        break;
+    }
+  }
+
   render() {
     const { items, count, errorMessage, errorStatus, filters } = this.state;
-    const { classes, onSelect, onAdd, t, keycloak, paginationMode = '' } = this.props;
+    const { classes, onSelect, onAdd, onDelete, t, keycloak, paginationMode = '' } = this.props;
+    const deleteLabel = t('common.delete');
+
+    const Actions = ({ item }) =>
+      onDelete ? (
+        <ConfirmationDialogTrigger
+          onCloseDialog={action => this.handleConfirmationDialogAction(action, item)}
+          dialog={{
+            title: t('entities.conference.deleteDialog.title'),
+            description: t('entities.conference.deleteDialog.description'),
```

Comment:
Maybe worth having these as common with placeholder variables?

Reply:
I expect those values to be specific to conference